Cargo.toml

```diff
@@ -1,9 +1,13 @@
 [package]
 name = "lm3s6965evb"
 version = "0.1.0"
-authors = ["Jorge Aparicio <jorge@japaric.io>"]
+authors = ["Jorge Aparicio <jorge@japaric.io>", "Hideki Sekine <sekineh@me.com>"]
+
+[features]
+semihosting = ["cortex-m-semihosting"]
```

Comment:
I don't think it's necessary to add a new feature. This repo is just a example; having it print to the console and exit by default seems fine to me. Could you remove the feature and always do the semihosting stuff?

Reply:
done this.